docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +259,29 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+You can view logs, traces, and metrics in Dynatrace dashboards:
+
+- To view the status of the Dynatrace integration with the Kyma Telemetry module, go to Dashboards > Import Dashboard, and manually import the file [Telemetry Module Status](./telemetry-resource-metrics.json).
+    > [!WARNING]
+    > Kyma Telemetry resources contain the following custom OpenTelemetry resource attributes:
+    > - `k8s.resource.name`
+    > - `k8s.resource.group`
+    > - `k8s.resource.kind`
+    > - `k8s.resource.version`
+    > 
+    > For the dashboard to function properly, these attributes must be added to the allow list of OpenTelemetry metrics resource attributes. To make these configuration changes, go to Settings > Metrics > OpenTelemetry metrics > Allow list: resource and scope attributes.
```

Comment:
can we use active voice?

```suggestion
    > For the dashboard to function properly, you must add these attributes to the allow list of OpenTelemetry metrics resource attributes. To make these configuration changes, go to Settings > Metrics > OpenTelemetry metrics > Allow list: resource and scope attributes.
```

Where do I make these configuration changes? In the Dynatrace UI? If that's the case, again I'd caution against documenting someone else's UI on click-level. Instead, we should link to their doc where they describe how to add something to the allowlist.